spock-core/src/main/java/org/spockframework/mock/runtime/ByteBuddyMockFactory.java

```diff
@@ -83,6 +83,26 @@ class ByteBuddyMockFactory {
     }
   }
 
+  /**
```

Comment:
Could you please add a line in the `2.4 (tdb) Misc` section that the issue #2017 was fixed.

See file: `docs/release_notes.adoc`